cmd/argocd-server/commands/argocd_server.go

```diff
@@ -151,12 +153,20 @@ func NewCommand() *cobra.Command {
 			stats.StartStatsTicker(10 * time.Minute)
 			stats.RegisterHeapDumper("memprofile")
 
+			var closer func()
 			for {
 				ctx := context.Background()
 				ctx, cancel := context.WithCancel(ctx)
+				if otlpAddress != "" {
+					closer, err = traceutil.InitTracer(ctx, "argocd-server", otlpAddress)
+					if err != nil {
+						log.Fatalf("failed to initialize tracing: %v", err)
+					}
+				}
```

Comment:
The same needs to be done for the repo_server and cmp_server commands

Reply:
What did you mean by `same`? Do you mean I need to start them using a for loop similar to argocd server? I think their command structures are different.